deploy_and_run_tests.sh

```diff
@@ -226,13 +227,14 @@ python gcp_variant_transforms/testing/integration/run_vcf_to_bq_tests.py \
     --logging_location "gs://${gs_dir}/temp/logs" \
     --image "${full_image_name}" ${TEST_ARGUMENTS}
 
-if  [[ -n "${run_preprocessor_tests}" ]]; then
-python gcp_variant_transforms/testing/integration/run_preprocessor_tests.py \
-    --project "${project}" \
-    --staging_location "gs://${gs_dir}/staging" \
-    --temp_location "gs://${gs_dir}/temp" \
-    --logging_location "gs://${gs_dir}/temp/logs" \
-    --image "${full_image_name}" ${TEST_ARGUMENTS}
+if [[ -n "${run_preprocessor_tests}" ]]; then
+  pip install --upgrade google-cloud-storage
```

Comment:
Suggestion: Instead of doing a specific pip install here, please move this dependency to the `setup.py` file. There is a section in that file that has integration test specific dependencies, i.e., `int_test`. You can see that in this very script, at line 220 above, we actually do `pip install --upgrade .[int_test]` and that should take care of any dependency that is int_test specific.